grpc/src/main/proto/model/taskdef.proto

```diff
@@ -10,7 +10,8 @@ option go_package = "github.com/netflix/conductor/client/gogrpc/conductor/model"
 message TaskDef {
     enum RetryLogic {
         FIXED = 0;
-        EXPONENTIAL_BACKOFF = 1;
+        LINEAR_BACKOFF = 1;
```

Comment:
changing the ordinal of the enum fields here would have unforeseen consequences for users using the grpc server and proto classes. A possible fix would be to append the new value at the end of the enum in TaskDef

Reply:
Adjusted the enum and the `taskdef.proto` is fixed (automatically) as well..